### DISCUSSION TITLE
Use cases

### DISCUSSION THREAD

**Discussion by dhruvarora2** [tobiasehlert/teslamateapi#81] [General]
Hi, 
The code looks good! Just want to know some use cases for this? Do you plan on making an APP to use these API's?

**Comment by tobiasehlert** [comment]
Hello @dhruvarora2,

I built this API to be able to get data into other systems by adding a RestAPI to TeslaMate. I use it mostly for home automation, but the possibilities are endless :-)

Two examples for a nice automation you could do:
1. If car status is driving and both my wife's and my cellphone have disconnected from the WiFi network, turn lighting into away-mode.
2. If it's 23:00 in the evening, the battery has less than 35% of charge and the charger is not connected, send notification to plug in the car (my wife forgets it when coming home).

And regarding the question about an app, no I don't have the intention to write an app for that. I had originally planned to write a WearOS app to get details about the car in my Skagen Falster 3, but those plans are currently on ice :-)

Regards,
Tobias
